### PR TITLE
Enable single daily by default + update tests

### PR DESCRIPTION
**Pull request by g-duval**

### CODE REVIEW

File: aggregator/src/main/java/com/redhat/cloud/notifications/config/AggregatorConfig.java [hunk 1]
```diff
@@ -34,7 +34,7 @@ private static String toggleName(String feature) {
     @Deprecated(forRemoval = true, since = "To be removed when we're done migrating to Unleash in all environments")
     boolean unleashEnabled;
 
-    @ConfigProperty(name = "notifications.bundle.level.digest.enabled", defaultValue = "false")
+    @ConfigProperty(name = "notifications.bundle.level.digest.enabled", defaultValue = "true")
```
Comment (gwenneg) [inline aggregator/src/main/java/com/redhat/cloud/notifications/config/AggregatorConfig.java:37]: Could you please update the ClowdApp template as well?

File: engine/src/main/java/com/redhat/cloud/notifications/config/EngineConfig.java [hunk 2]
```diff
@@ -44,7 +44,7 @@ public class EngineConfig {
     @Deprecated(forRemoval = true, since = "To be removed when we're done migrating to Unleash in all environments")
     boolean asyncAggregation;
 
-    @ConfigProperty(name = "processor.email.aggregation.use-recipients-resolver-clowdapp.enabled", defaultValue = "false")
+    @ConfigProperty(name = "processor.email.aggregation.use-recipients-resolver-clowdapp.enabled", defaultValue = "true")
```
Comment (gwenneg) [inline engine/src/main/java/com/redhat/cloud/notifications/config/EngineConfig.java:47]: This looks unrelated and could be a breaking change in some environments.

File: aggregator/src/main/java/com/redhat/cloud/notifications/config/AggregatorConfig.java (hunk 1 above)
Comment (g-duval) [reply aggregator/src/main/java/com/redhat/cloud/notifications/config/AggregatorConfig.java:37]: Sure

File: engine/src/main/java/com/redhat/cloud/notifications/config/EngineConfig.java (hunk 2 above)
Comment (g-duval) [reply engine/src/main/java/com/redhat/cloud/notifications/config/EngineConfig.java]: That's the idea, but I suppose it have to be validated on dedicated stage env before pomoting it to prod. May by I'm expecting too much, let's rollback it ;)

**Review by gwenneg — APPROVED** [review]
LGTM
